api/app/utils/review.inference.ts

```diff
@@ -59,6 +56,14 @@ const buildIsExpiredAndNotArchivedCondition = (currentTime: number) => {
   };
 };
 
+const buildIsExpiredAndArchivedOrNotArchivedCondition = (currentTime: number) => {
```

Comment:
For simplicity I would call this `buildisExpired` cause that's what it is afterall
And maybe a comment explaining why there is this need for this logic (kind of what I explained in my prev comment)